.github/workflows/ci.yml

```diff
@@ -40,7 +40,7 @@ jobs:
           - '2.13.14'
         java:
           - '8'
-          - '21'
+          - '17'
```

Comment:
is this part of this change?

Reply:
Yeah, id doesn't compile for 21 on older Scala 2 versions